src/chain/member_info.rs

```diff
@@ -7,18 +7,40 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 /// Information about a member of our section.
+
+pub type AgeCounter = u32;
+
+/// The minimum allowed value of the Age Counter
+pub const MIN_AGE: AgeCounter = 16;
+
+const MAX_INFANT_AGE: u8 = 4;
+
 #[derive(Clone, Copy, Eq, PartialEq, Hash, Serialize, Deserialize, Debug)]
 pub struct MemberInfo {
     pub persona: MemberPersona,
-    pub age: u8,
+    pub age_counter: AgeCounter,
     pub state: MemberState,
 }
 
+impl MemberInfo {
+    pub fn age(self) -> u8 {
+        f64::from(self.age_counter).log2() as u8
+    }
+
+    #[allow(unused)]
+    pub fn increase_age(&mut self) {
+        self.age_counter += 1;
+        if self.age() > MAX_INFANT_AGE && self.persona == MemberPersona::Infant {
```

Comment:
Can we do `self.age_counter > 2.pow(MAX_INFANT_AGE)` to avoid relying on floating point arithmetic?